src/Microsoft.VisualBasic/src/Microsoft/VisualBasic/Helpers/NativeMethods.vb

```diff
@@ -0,0 +1,241 @@
+﻿' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+Option Explicit On
+Option Strict On
+
+Imports System.Runtime.InteropServices
+Imports System.Runtime.Versioning
+Imports System.Security
+
+Namespace Microsoft.VisualBasic.CompilerServices
+
+    <ComVisible(False)>
+    Friend NotInheritable Class NativeMethods
+        <SecurityCritical()>
+        <ResourceExposure(ResourceScope.None)>
+        <PreserveSig()>
+        Friend Declare Function _
+            CloseHandle _
+                Lib "kernel32" (ByVal hObject As IntPtr) As Integer
+
+        <DllImport("user32.dll", CharSet:=CharSet.Unicode, EntryPoint:="MessageBox", SetLastError:=False)>
+        Friend Shared Function MessageBox(hWnd As Integer, lpText As String, lpCaption As String, uType As UInteger) As Integer
+        End Function
```

Comment:
Is `MessageBox` used?

Reply:
It was used in testing and I moved it there.